WeakAuras/Prototypes.lua

```diff
@@ -7012,26 +7066,54 @@ Private.event_prototypes = {
       ]];
         ret = ret..ret2:format(tonumber(trigger.remaining or 0) or 0);
       end
-      return ret:format(trigger.rune, "[[" .. (trigger.genericShowOn or "") .. "]]");
+      if WeakAuras.IsWrathClassic() then
+        return ret:format(trigger.rune, (trigger.use_inverse and "true" or "false"), (trigger.use_deathRune == true and "true" or trigger.use_deathRune == false and "false" or "nil"), (trigger.use_includeDeath and "true" or "false"));
+      else
+        return ret:format(trigger.rune, "[[" .. (trigger.genericShowOn or "") .. "]]");
+      end
     end,
     args = {
       {
         name = "rune",
         display = L["Rune"],
         type = "select",
         values = "rune_specific_types",
-        test = "(genericShowOn == \"showOnReady\" and (startTime == 0)) " ..
+        test = WeakAuras.IsWrathClassic() and [[
+          ((inverse and startTime == 0) or (not inverse and startTime > 0))
+          and
+          ((death == nil) or (death == true and GetRuneType(rune) == 4) or (death == false and GetRuneType(rune) ~= 4))
+        ]] or "(genericShowOn == \"showOnReady\" and (startTime == 0)) " ..
         "or (genericShowOn == \"showOnCooldown\" and startTime > 0) "  ..
         "or (genericShowOn == \"showAlways\")",
-        enable = function(trigger) return not trigger.use_runesCount end,
+        enable = function(trigger)
+          if WeakAuras.IsWrathClassic() then
+            return not trigger.use_bloodRunes and not trigger.use_unholyRunes and not trigger.use_frostRunes
+          else
+            return not trigger.use_runesCount
+          end
+        end,
         reloadOptions = true
       },
+      {
+        name = "deathRune",
+        display = L["Death Rune"],
+        type = "tristate",
+        test = "true",
+        enable = function(trigger) return WeakAuras.IsWrathClassic() and trigger.use_rune end
```

Comment:
hidden field should be set if WeakAuras.IsWrathClassic() is false for all wotlk specific options

Reply:
Added to each of the wotlk options.